cypress/e2e/3-testing/loginCorrectInfo.cy.js

```diff
@@ -0,0 +1,30 @@
+// login  credentials
+// name 	vegar
+// email 	no@noroff.no
+// pass 	pass123123
+
+// Describe the function
+
+describe("login correct credentials", () => {
+  it("should log into the Social Media platform", () => {
+    // We need to go to the index.html page
+    cy.visit("http://127.0.0.1:5500/");
+
+    // Then the login form and enter credentials
+    cy.get("[data-cy='emailInp']")
```

Comment:
This attribute does not exist in the original application. You are not permitted to edit the original code in this assignment as the purpose is to simulate working on an existing product.